applications/GeoMechanicsApplication/custom_elements/interface_stress_state.h

```diff
@@ -0,0 +1,38 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#pragma once
+
+#include "stress_state_policy.h"
+
+namespace Kratos
+{
+
+class KRATOS_API(GEO_MECHANICS_APPLICATION) InterfaceStressState : public StressStatePolicy
+{
+public:
+    [[nodiscard]] Matrix CalculateBMatrix(const Matrix&         rDN_DX,
+                                          const Vector&         rN,
+                                          const Geometry<Node>& rGeometry) const override;
+    [[nodiscard]] double CalculateIntegrationCoefficient(const Geometry<Node>::IntegrationPointType& rIntegrationPoint,
+                                                         double DetJ,
+                                                         const Geometry<Node>& rGeometry) const override;
+    [[nodiscard]] Vector CalculateGreenLagrangeStrain(const Matrix& rDeformationGradient) const override;
+    [[nodiscard]] std::unique_ptr<StressStatePolicy> Clone() const override;
+    [[nodiscard]] const Vector&                      GetVoigtVector() const override;
+    [[nodiscard]] SizeType                           GetVoigtSize() const override;
+    [[nodiscard]] SizeType                           GetStressTensorSize() const override;
+    static const Vector                              VoigtVectorInterface2D;
```

Comment:
Since this is an implementation detail, let's make this `static` data member `private`.

Reply:
It compiles, so done!